invoices/interface.go

```diff
@@ -198,3 +199,32 @@ type InvoiceUpdater interface {
 	// Finalize finalizes the update before it is written to the database.
 	Finalize(updateType UpdateType) error
 }
+
+// InterceptClientRequest is the request that is passed to the client via
+// callback during an interceptor session. The request contains the invoice that
+// is being intercepted and supporting information.
+type InterceptClientRequest struct {
+	// ExitHtlcCircuitKey is the circuit key that identifies the HTLC which
+	// is involved in the invoice settlement.
+	ExitHtlcCircuitKey CircuitKey
+
+	// Invoice is the invoice that is being intercepted.
+	Invoice Invoice
+}
```

Comment:
don't we want to add more data here? I'd imagine that for example in the tap channels case, you'd want to provide any custom blobs that would need to be checked here

Reply:
I think what precicely we need here will become more clear once we actually start using the `InvoiceAcceptor` from tapd.